### PR TITLE
Add total voxel count and dataset extent to tooltip in dataset info tab

### LINKED ISSUE #8128: Tooltip with voxel count
## Detailed Description
Add a tooltip that shows the total number of voxels in a dataset (i.e. the product of the dataset extent) in voxels and in the physical unit (ie. the volume of the dataset).

## Context
The tooltip could be shown on hover here:
Screenshot 2024-10-16 at 16 12 33

### PR DESCRIPTION
**Pull request by dieknolle3333**
Screenshot from 2024-11-25 14-39-07

### URL of deployed dev instance (used for testing):
- https://dsextenttooltip.webknossos.xyz/

### Steps to test:
- Open any annotation or DS in view mode
- go to dataset info tab
- hover above icon next to factorized dataset extent in unit and in vx
- read the product of the extent in vx and unit :) 

### TODOs:
- [x] make sure it looks good with large numbers, use scientific notation for large numbers? (found method to format vx)

### Issues:
- fixes #8128 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

**Comment by dieknolle3333** [comment]
Screenshot from 2024-11-25 14-04-53
for larger datasets

File: frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts [hunk 1]
```diff
@@ -345,6 +345,19 @@ export function getDatasetExtentAsString(
     formatNumberToLength(length, LongUnitToShortUnitMap[dataset.dataSource.scale.unit]),
   );
 }
+function getDataSetExtentAsProduct(extent: {
+  width: number;
+  height: number;
+  depth: number;
+}) {
+  return extent.width * extent.height * extent.depth;
+}
```
[the remark this answers is not in the document]
Comment (dieknolle3333) [reply frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts:354]: I feel like 1. and 2. are valid points!

File: frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts [hunk 2]
```diff
@@ -345,6 +345,34 @@ export function getDatasetExtentAsString(
...
+  height: number;
+  depth: number;
+}) {
+  const safeMultiply = (a: number, b: number) => {
+    const result = a * b;
+    if (!Number.isFinite(result)) {
+      throw new Error("Dataset extent product exceeds maximum safe number");
+    }
+    return result;
+  };
```
Comment (philippotto) [inline frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts:359]: hmmm, I'm not really fond of this. as a preface, I think, that it's unrealistic that we will ever run into this (maximum number in JS is 2^1024 which means that one dimension of a dataset would roughly have to have 2^342 voxels at least for this to be triggered). so, I wouldn't want to clutter the code here with that.

my suggestion would be: remove the safeMultiply + try-catch stuff here in this module and then simply ensure that `formatNumberToVolume` and `formatVoxels` can deal with `Infinity` as an input (not sure what it does currently).

File: frontend/javascripts/oxalis/view/right-border-tabs/dataset_info_tab_view.tsx [hunk 3]
```diff
@@ -127,9 +129,27 @@ const shortcuts = [
 export function DatasetExtentRow({ dataset }: { dataset: APIDataset }) {
   const extentInVoxel = getDatasetExtentAsString(dataset, true);
   const extentInLength = getDatasetExtentAsString(dataset, false);
+  const extentProductInVx = getDatasetExtentInVoxelAsProduct(dataset);
+  const extentProductInUnit = getDatasetExtentInUnitAsProduct(dataset);
+  const formattedExtentinUnit = formatNumberToVolume(
```
Comment (philippotto) [inline frontend/javascripts/oxalis/view/right-border-tabs/dataset_info_tab_view.tsx:134]: `in` -> `In`

**Review by philippotto — APPROVED** [review]
lgtm! only see my one nitpick before merging.

**Pull request merged**